base_layer/core/src/range_proof.rs

```diff
@@ -26,6 +26,6 @@
 const RANGE_PROOF_LENGTH: usize = 1; // This will be changed
 
 #[derive(Debug, Clone)]
-pub struct RangeProof([u8; RANGE_PROOF_LENGTH]);
+pub struct RangeProof(pub [u8; RANGE_PROOF_LENGTH]);
```

Comment:
What's the reason for exposing this as `pub`?

Reply:
mainly just for convenience in the interim, this struct will be replaced by a real implementation soon enough.